src/cbuild/hooks/post_install/007_strip_debug.py

```diff
@@ -159,5 +161,59 @@ def invoke(pkg):
     else:
         (pkg.destdir / "usr/lib").rmdir()
 
+    # now collect the referenced debug sources
+    for v in ddest.rglob("*"):
+        if v.is_dir():
+            continue
```

Comment:
thought of this while on the train-

the strip currently was made some amount faster by being able to do the intermediate strip with just `llvm-strip <every file>` instead of once per file. llvm-dwarfdump also supports multiple inputs, instead of only one at a time.

you should check what happens with

```
llvm-dwarfdump --show-sources /package/file/1
llvm-dwarfdump --show-sources /package/file/2
llvm-dwarfdump --show-sources /package/file/1 /package/file/2
```

does it work (and just return all the source lists combined)? or, even better, perhaps automatically dedupe them so we have fewer shutil copies below for the same referenced files (since usually the same headers are copied over and over again, etc)?

but as long as it works correctly (we copy the same source files; the return of the command is the same total output), doing this in one call is just better; so use this `for v` loop to build a list of sources with the v.relative_to transform, then pass it at once with `*mylist` instead, of doing it one at a time.

Reply:
answering my own question; it works, and the lists are concatenated together with no dedup